core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/MockHttpGoogleServicesDAO.scala

```diff
@@ -68,18 +70,33 @@ class MockHttpGoogleServicesDAO(
     credential
   }
 
-  protected override def listBillingAccounts(credential: Credential)(implicit executionContext: ExecutionContext): Future[List[BillingAccount]] = {
-    val firecloudHasThisOne = new BillingAccount()
-      .setDisplayName("testBillingAccount")
-      .setName(accessibleBillingAccountName.value)
-      .setOpen(true)
-
-    val firecloudDoesntHaveThisOne = new BillingAccount()
-      .setDisplayName("testBillingAccount")
-      .setName(inaccessibleBillingAccountName.value)
-      .setOpen(true)
-
-    Future.successful(List(firecloudHasThisOne, firecloudDoesntHaveThisOne))
+  protected override def executeGoogleListBillingAccountsRequest(credential: Credential, pageToken: Option[String] = None)(implicit counters: GoogleCounters): ListBillingAccountsResponse = {
```

Comment:
`listBillingAccounts` calls `executeGoogleListBillingAccountsRequest`, so now I am testing the guts of `listBillingAccounts` to make sure it handles pagination. At the same time, adding coverage that non-open billing accounts are filtered out.